### DISCUSSION TITLE
DSLR to HDMI to Raspberry Pi

### DISCUSSION THREAD

**Discussion by Groock** [jim-easterbrook/python-gphoto2#154] [Q&A]
Hi I'm working on a digital art-research PhD project

I was wondering if it's possible to use gphoto2 to access the DSLR via HDMI cable?

I'm writing a python GUI program with Tkinter and Opencv2 libraries for a physical device that I made with a raspberry pi/ touch screen/ USB camera, but the USB camera really needs the zoomable quality of the DSLR that I could attach to the device (via HDMI cable).

Really appreciate this library and its GNU license

**Comment by jim-easterbrook** [comment]
I've absolutely no idea. As this is a general question about gphoto2 you should ask on the gphoto2 mailing list. Someone there may have used HDMI for camera control. (I didn't know HDMI could do that, I thought it was only for video streaming.)
